modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ zephyr_library()
 zephyr_include_directories(.)
 zephyr_library_sources_ifdef(CONFIG_TFM_ALLOW_NON_SECURE_FAULT_HANDLING fault.c)
 
-if (CONFIG_TFM_PARTITION_PLATFORM AND CONFIG_SOC_FAMILY_NORDIC_NRF)
+if(CONFIG_TFM_PARTITION_PLATFORM AND CONFIG_SOC_FAMILY_NORDIC_NRF)
```

Comment:
I can see lots of these whitespace changes. Can you try to re-edit this file so that your editor does not touch any other lines than ones you change.

Reply:
Whitespace changes separated to different commit. 

See commit https://github.com/nrfconnect/sdk-nrf/pull/17815/commits/f6d6888658f4012f000bd8f07907f4d47e456486 for the functional changes.